es9/2018-05/summary.md

```diff
@@ -0,0 +1,80 @@
+# Summary for the 64th meeting of Ecma TC39
+
+Ecma Technical Committee 39 held three day meeting at Bloomberg in NYC on May 22nd - May 24th, 2018. [Agenda can be found here](https://github.com/tc39/agendas/blob/master/2018/05.md) (note: committee has adopted new agenda formart based on the stage of proposals)
+
+## New Proposals
+TC39 discussed 10 new proposals. 7 entered staging process to discuss towards specification, and 3 remains at Stage 0 for further investigation before consensus.  
+
+### Advanced to Stage 1 
+- [Sequence properties in Unicode property escapes](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-22.md#11ia-sequence-properties-in-unicode-property-escapes)
+- [Module Keys](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-23.md#module-keys-strawman-for-stage-1)
+- [Class Static Block](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-23.md#class-static-block)
+- [Class Access Expressions](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-23.md#class-access-expressions)
+- [Pattern Matching](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-24.md#pattern-matching-for-stage-1)
+- [RegExp Match array offsets](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-24.md#bl%C3%B6cks-syntax-for-stage-0)
+
+### Advanced to Stage 2 
+- [Well-formed JSON.stringify](https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-22.md#stringprototypecodepoints-for-stage-2)
```

Comment:
the content is great, but we can shorten the urls in this document, basically replacing:

```
https://github.com/rwaldron/tc39-notes/blob/master/es9/2018-05/may-22.md
```

with only

```
may-22.md
```

same for may-23.md and may-24.md.